blog.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial=scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Blog</title>
+  </head>
+  <body>
+    <header>
+      <a href="index.html">
+        <img src="images/adoptadog-logo.svg" alt="Adopt a Dog" width="300">
```

Comment:
Remember to keep the alt tags to be descriptive to what the image is. If it's a logo, it's maybe ok just to use Adopt a dog logo or something.

For now the width attribute is fine, but when you start folding in CSS files you'll want them all out.